content/2016.12.how-to-develop-a-webapp-with-Kinto.md

```diff
@@ -0,0 +1,256 @@
+Title: How to develop a WebApp with Kinto
+Date: 2016-12-01 10:20
+Slug: develop-a-chat-webapp-with-kinto
+Authors: Enguerran Colson, Frank Taillandier, Kinto Team
+Summary: An example of how Kinto handles a webapp's data storage.
+
+_This post was [originally published in french on Enguerran Colson's blog](https://blog.ticabri.com/2016/11/29/developper-une-webapp-avec-kinto/)._
+
+## Our goal
+
+Hi folks, today, we are going to create a simple WebApp that allows visitors to chat in a room accessible from a public URL.
+
+![KintoChat, our instant chat WebApp](images/kintochat-fullscreen.png)
+{.image}
+
+What will we need?
+
+* A really simple chat webapp with minimal UI.
+* We will need a bit of JavaScript to enhance and animate interactions within the application and we will also need a way to store and share the app's data. Ideally, we would like our application to consume a REST API exposing endpoints like `https://api.discute.re/v1/rooms/12345/messages`, where `discute.re` is the domain of our application (*discutere* means to chat in Latin).
+
+Our API endpoints would be:
+
+* `/rooms` to return all chat rooms,
+* `/rooms/:id/` to return a particular chat room identified by an ID,
+* `/rooms/:id/messages` to return all messages from a particular room.
+
+To post a message, our front-end will do a POST request to `/rooms/:id/messages`.
+
+We would store each message like this:
+
+```json
+{
+    "message": "I know nothing",
+    "last_modified": 1480340246147,
+    "author": "Jon Snow",
+    "id": "a1e6e879-5c94-4a01-9ec2-28e152d1d8f4"
+}
+```
+
+The JavaScript code could use whatever library or latest hype JS framework but in our example we will stick with modern [Vanilla JS](http://vanilla-js.com/) and use top notch [ES2015 syntax](https://babeljs.io/docs/learn-es2015/).
```

Comment:
Different dictionaries say different things but both "stick to" and "stick with" seem pretty much the same to me in this context. :)